crates/shim/proto/vm_service.proto

```diff
@@ -41,6 +41,11 @@ message GetFileRequest {
   string path = 2;
 }
 
+message File {
+  string path = 1;
+  bytes checksum = 3;
+}
+
```

Comment:
Ok I change that.

Reply:
In fact it doesn't work because the checksum is calculated faster the `FileMetadata` has been sent. I use the chunk slicing of submit_file to calculate the checksum. It avoids reading all the file only for the checksum.